rest_framework/compat.py

```diff
@@ -140,6 +140,16 @@ def value_from_object(field, obj):
     return field.value_from_object(obj)
 
 
+def getargspec(obj):  # type: tuple
+    if not hasattr(inspect, 'signature'):
```

Comment:
Also the `# type: tuple` seems redundant - if we're going to describe the return type could we instead either do so more fully or not at all.